src/components/scenes/FioRequestListScene.tsx

```diff
@@ -352,14 +354,18 @@ class FioRequestList extends React.Component<Props, LocalState> {
   }
 
   renderDropUp = async (selectedFioPendingRequest: FioRequest) => {
-    const { onSelectWallet } = this.props
+    const { account, onSelectWallet } = this.props
     const { content } = selectedFioPendingRequest
-    const chainCode = content.chain_code.toUpperCase()
-    const tokenCode = content.token_code.toUpperCase()
-    const allowedFullCurrencyCode: string[] = chainCode !== tokenCode && tokenCode && tokenCode !== '' ? [`${chainCode}-${tokenCode}`] : [chainCode]
+    const pluginId = Object.keys(SPECIAL_CURRENCY_INFO).find(
+      pluginId => SPECIAL_CURRENCY_INFO[pluginId].fioChainCode ?? SPECIAL_CURRENCY_INFO[pluginId].chainCode === content.chain_code.toUpperCase()
+    ) as string
```

Comment:
This might return `undefined`, which is why you have an `as string` on the end. However, I think it might be better to do some sort of `if (pluginId == null) showError("Oy, mate! We don't got this one.")`

Reply:
Oof yeah good catch